src/Report.php

```diff
@@ -21,7 +21,7 @@ class Report implements ReportInterface
     protected $uri;
 
     /**
-     * @var integer
+     * @var \DateTime
```

Comment:
Is that legit? If so, please consider creating a different PR with this as a bugfix instead of part of this PR. Thanks!